core/storage/classifier/gae_models.py

```diff
@@ -144,12 +146,13 @@ def _generate_id(cls, exp_id):
 
     @classmethod
     def create(
-            cls, algorithm_id, exp_id, exp_version, training_data,
+            cls, algorithm_id, committer_id, exp_id, exp_version, training_data,
             state_name, status=feconf.TRAINING_JOB_STATUS_NEW):
         """Creates a new ClassifierTrainingJobModel entry.
 
         Args:
             algorithm_id: str. ID of the algorithm used to generate the model.
+            committer_id: str. ID of the user who created the model.
```

Comment:
This is not the person who created the model. The model is created as a consequence of his action of storing new exploration version.